planner/core/logical_plan_builder.go

```diff
@@ -2691,13 +2691,13 @@ func (b *PlanBuilder) buildUpdate(ctx context.Context, update *ast.UpdateStmt) (
 	return updt, err
 }
 
-func (b *PlanBuilder) buildUpdateLists(ctx context.Context, tableList []*ast.TableName, list []*ast.Assignment, p LogicalPlan) ([]*expression.Assignment, LogicalPlan, error) {
+func (b *PlanBuilder) buildUpdateLists(ctx context.Context, tableList []*ast.TableName, list []*ast.Assignment, p LogicalPlan) ([]*expression.Assignment, LogicalPlan, bool, error) {
```

Comment:
The function signature is a little too long, how about splitting it to several lines:
```go
func (b *PlanBuilder) buildUpdateLists(
	ctx context.Context,
	tableList []*ast.TableName,
	list []*ast.Assignment,
	p LogicalPlan,
) (
	[]*expression.Assignment,
	LogicalPlan,
	bool, error,
) {
```